react/src/components/BookList/index.js

```diff
@@ -2,29 +2,41 @@ import React, { Component } from 'react';
 
 import BookItem from './components/BookItem';
 import {data} from '../../mock.js';
+import booksGet from '../../services/booksGet';
 
 import './styles.css';
 
 class BookList extends Component {
+  state = {data: null};
+  componentWillMount(){
+    booksGet(
+      sessionStorage.getItem('access_token')
```

Comment:
Once the login is successful, I'd set the API header with this value so we avoid putting it every time we fetch data from the backend